docker/py3-tools-ubi/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "py3-tools-deb"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+cloudshare = "*"
+intezer-sdk = "*"
+emoji = "*"
+exifread = "*"
+"stomp.py" = "*"
+geopy = "*"
+dnspython = "*"
+algorithmia = "*"
+asana = "*"
+feedparser = "*"
+beautifulsoup4 = "*"
+mail-parser = "*"
+imap-tools = "*"
+imapclient = "*"
+pyjarm = "*"
+pyminizip = "*"
+paho-mqtt = "*"
+arrow = "*"
+pytz = "*"
+python-dateutil = "*"
+vertica-python = "*"
+sendgrid = "*"
+circleci = "*"
+rsa-archer = "*"
+tweepy = "*"
+tld = "*"
+validate-email = "*"
+requests = "*"
+markdownify = "*"
+langdetect = "*"
+gql = "*"
+minio = "*"
+ldap3 = "*"
+pymongo = "*"
+cyjax-cti = "*"
+pyzabbix = "*"
+jmespath = "*"
+tldextract = "*"
+stix2 = "*"
+taxii2-client = "*"
+xlrd = "==1.2.0"
```

Comment:
why is this locked? Can we leave a comment here (with a link to issue to unlcock if relevant

Reply:
I don't know, i copied it directly from https://github.com/demisto/dockerfiles/blob/master/docker/py3-tools/Pipfile#L47